src/main/java/step2/domain/LottoNumber.java

```diff
@@ -0,0 +1,70 @@
+package step2.domain;
+
+import step2.exception.LottoException;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+    public static final int MIN_LOTTONUMBER = 1;
+    public static final int MAX_LOTTONUMBER = 45;
+    private final int number;
```

Comment:
상수와 인스턴스 변수의 구분을 위해서 공백 라인을 추가해보시면 좋을 것 같습니다.
```suggestion
    public static final int MAX_LOTTONUMBER = 45;
    
    private final int number;
```